package.json

```diff
@@ -30,7 +30,7 @@
     "browserify-zlib": "~0.2.0",
     "buffer": "^5.0.2",
     "cached-path-relative": "^1.0.0",
-    "concat-stream": "^1.6.0",
+    "concat-stream": "^2.0.0",
```

Comment:
What is the breaking change here?

Reply:
Fair point; I have been cleansing a lot of npm code and this change probably came on the heels of another package I updated to 2.0.0. IMO it's no harm, no foul to keep it at 1.6.0; and it's no-harm, no foul to tick it up. I am resubmitting a PR without the concat-stream 2.0.0